samples/iterable_dataset/dataset_from_blob_list.py

```diff
@@ -0,0 +1,21 @@
+"""
+This sample demonstrates the way to create an iterable-style dataset from a blob list using the IterableBlobDataset class.
+"""
```

Comment:
For this docstring and the other docstrings, we can probably shorten the prose even more and make it fit to a single line. For example:
```
"""Creates an iterable-style dataset from a list of blobs"""
```
As a rule of thumb, if the docstring is a one-liner, the recommendation is to include double quotes on the same line.